package.json

```diff
@@ -2,9 +2,10 @@
   "name": "jslearn-complited-tasks",
   "version": "1.0.0",
   "description": "",
-  "main": "./src/modules/stack.js",
+  "main": "./src/lib/stack/stack.js",
```

Comment:
you package is list of libraries, so you shouldn't specify one certain module as "main", use here just "main": "index.js". Later there will be a list of exported lib's.

Reply:
ok, i added index.js to /src